mkosi/__init__.py

```diff
@@ -6581,11 +6581,11 @@ def load_args(args: argparse.Namespace) -> MkosiArgs:
         args.output_sshkey = args.output.with_name("id_rsa")
 
     if args.split_artifacts:
-        args.output_split_root = build_auxiliary_output_path(args, ".usr" if args.usr_only else ".root", True)
+        args.output_split_root = build_auxiliary_output_path(args, ".usr.raw" if args.usr_only else ".root.raw", True)
         if args.verity:
-            args.output_split_verity = build_auxiliary_output_path(args, ".verity", True)
+            args.output_split_verity = build_auxiliary_output_path(args, ".usr.verity" if args.usr_only else ".root.verity", True)
             if args.verity == "signed":
-                args.output_split_verity_sig = build_auxiliary_output_path(args, ".verity-sig", True)
+                args.output_split_verity_sig = build_auxiliary_output_path(args, ".usr.usrhash.p7s" if args.usr_only else ".root.roothash.p7s", True)
```

Comment:
```suggestion
                args.output_split_verity_sig = build_auxiliary_output_path(args, f"{roothash_suffix(args}.p7s", True)
```
and we could possibly make some
```python
def root_or_usr(args):
    return ".usr" if args.usr_only else ".root
```
to shorten all of these similar to the above suggestion.